src/App.vue

```diff
@@ -190,6 +201,16 @@ export default class App extends Mixins(StateMixin) {
       this.flashMessage.timeout = (payload && payload.timeout !== undefined) ? payload.timeout : undefined
       this.flashMessage.open = true
     })
+
+    const legacyFavIcons = document.querySelectorAll("link[rel*='icon'][type='image/png']")
```

Comment:
This bit here is kind of a "hack" that will remove the existing `link rel="icon"...` that is put in by the PWA stuff. We would end up with 2 entries and the icon would not update, so this will remove the ones from the generated index.html.